src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -42,11 +42,13 @@ def setUp(self):
     helpers.patch(self, [
         'crash_analysis.stack_parsing.stack_symbolizer.symbolize_stacktrace',
         'metrics.logs.log_error',
+        'platforms.android.settings.get_repo_prop_path'
     ])
 
     os.environ['JOB_NAME'] = TEST_JOB_NAME
 
     self.mock.symbolize_stacktrace.side_effect = self._mock_symbolize_stacktrace
```

Comment:
better to just do self.mock.symbolize_stacktrace.return_value = 'value' directly instead of defining a new function.

Reply:
Well you need to return the first element passed to it, I don't think you can do that with return_value, you could do a lambda I guess.  I copied this from the other stack_analyzer_test.py